tests/Validation/ReCaptchaValidationTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Rules\ReCaptchaValidation;
+
+class ReCaptchaValidationTest extends TestCase
+{
+    public function buildMockedReCaptchaRule($fakeResponse)
```

Comment:
Given we are supposed to be testing the RecaptchaRule its unfortunate that we need to mock parts of it to test it.

I was thinking about that we could inject the `\ReCaptcha\ReCaptcha` into the rule (see above comment on implementation of class). This would then enable us to mock just this class from the library to test similar functionality but also the logic of the `verify` method.